pkg/plugins/backendplugin/pluginextensionv2/buf.gen.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+plugins:
+  - plugin: go
+    out: pkg/plugins/backendplugin/pluginextensionv2
+  - plugin: go-grpc
+    out: pkg/plugins/backendplugin/pluginextensionv2
+    opt:
+      - require_unimplemented_servers=false
```

Comment:
```suggestion
      - require_unimplemented_servers=true
```

Reply:
nit: it's generally preferred to include the trailing newline at the end of the file

Reply:
I don't think `require_unimplemented_servers=true` makes any difference, but I added it thanks.